packages/xsnap/src/avaXS.js

```diff
@@ -253,8 +262,11 @@ async function avaConfig(args, options, { glob, readFile }) {
       case '--match':
         titleMatch = args.shift();
         break;
-      default:
-        files.push(arg);
+      default: {
+        // The argument is a glob for tests to run.
+        const argFiles = await globFiles(arg);
+        files.push(...argFiles);
+      }
```

Comment:
making avaXS more like ava is good stuff.

Especially when it only costs 2 or 3 lines :)